managed/services/config/pmm-managed.yaml

```diff
@@ -2,6 +2,7 @@ services:
   telemetry:
     enabled: true
     load_defaults: true
+    config_file_env: "PERCONA_TEST_TELEMETRY_FILE"
```

Comment:
what goal are we going to achieve by this? is it really will be used by users? Usually env vars defined in OS and this is not service responsibility to determine the way they are defined.

Reply:
as discussed offline, we can hardcode ENV var names in the code